beekeeper-core/src/main/java/com/expediagroup/beekeeper/core/repository/HousekeepingMetadataRepository.java

```diff
@@ -37,8 +37,9 @@
   @Query(value = "from HousekeepingMetadata t "
       + "where t.databaseName = :databaseName "
       + "and t.tableName = :tableName "
-      + "and t.partitionName = :partitionName "
+      + "and (t.partitionName = :partitionName or t.partitionName is NULL) " // To handle special null case
```

Comment:
This is a user facing change so could be worth mentioning unless we think it's such an edge case that it's not.

Reply:
It's part of the new stuff we have added so users aren't using this. It's just something I missed which was caught in the integration tests. Do you think I should add it to the CHANGELOG?

Reply:
No, if it's new and part of the broader changes which are already described in the CHANGELOG then it's fine.